mlflow/tracking/client.py

```diff
@@ -4729,5 +4729,5 @@ def print_model_version_info(mv):
             Aliases: ["test-alias"]
         """
         _validate_model_name(name)
-        _validate_model_alias_name(alias)
+        _validate_model_alias_name(alias, perform_latest_check=False)
```

Comment:
It does make sense to remove validation from this client codepath (and in `rest_store` etc if we have validation there). I think on reads just removing validation entirely is fine, we can let the backend throw on an invalid name etc. For removing client-side write validation, we should make sure analogous logic is present in all backends we maintain